core/modules/user/config/user.mail.json

```diff
@@ -1,6 +1,6 @@
 {
     "_config_name": "user.mail",
-    "cancel_confirm_body": "[user:name],\n\nA request to cancel your account has been made at [site:name].\n\nYou may now cancel your account on [site:url-brief] by clicking this link or copying and pasting it into your browser:\n\n[user:cancel-url]\n\nNOTE: The cancellation of your account is not reversible.\n\nThis link expires in one day and nothing will happen if it is not used.\n\n--  [site:name] team",
+    "cancel_confirm_body": "[user:name],\n\nA request to cancel your account has been made at [site:name].\n\nYou may now cancel your account on [site:url:brief] by clicking this link or copying and pasting it into your browser:\n\n[user:cancel-url]\n\nNOTE: The cancellation of your account is not reversible.\n\nThis link expires in one day and nothing will happen if it is not used.\n\n--  [site:name] team",
```

Comment:
Actually I take that back. Both the old and new tokens work fine. We can leave this as-is.